### PR TITLE
blog: Journey to Site Reliability Engineering at Gitpod

### PR DESCRIPTION
**Pull request by ArthurSens**
👋 First time writing a blog after almost a year, but I was never a blog writer type of person anyway :P

Any feedback appreciated, whether:
* Are images alright?
* Anything that should be reworded?
* Suggestions for better story-telling?

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/496"></a>

### CODE REVIEW

File: src/routes/blog/journey-to-site-reliability-engineering-at-gitpod.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+_We gave those cloud-based environments a unique name: "Workspaces"._
+
+Workspaces are the core of Gitpod, it is impossible to be a user of Gitpod without using Workspaces. To achieve the goal of making the world forget about local environments, _we need to be a service provider that developers can trust and rely on_, therefore, Workspaces need to be just as reliable as local environments!
+
+Gitpod is scaling! [We've recently announced that we got $$$$](https://www.gitpod.io/blog/next-chapter-for-gitpod/) and the topic "Cloud-based development" is on fire right now! Being the pioneers in this topic, it's only natural that we're getting more and more users every day. This is truly a great thing for us! We are honored to receive so much love in the last couple of months and we want to keep this momentum going, but this extra love also puts an extra load on top of our clusters and extra load sometimes leads to unreliability. It is awesome that people are starting to forget about their local environments thanks to Gitpod, therefore _we want Workspaces to maintain a fantastic performance, doesn't matter if we host ten, ten thousand, or ten million Workspaces concurrently._
```
Comment (ArthurSens) [inline src/routes/blog/journey-to-site-reliability-engineering-at-gitpod.md:24]: Ah please, how should I say that we have good investors? 😅

**Review by mads-hartmann — APPROVED** [review]
This is a great post 🎉

**Comment by ArthurSens** [comment]
Any extra thoughts @meysholdt @ghuntley @mikenikles?

If not, how do I proceed to get this blogpost into the website? 🤔

File: src/contents/blog.ts [hunk 2]
```diff
@@ -146,4 +146,13 @@ export const authors: { [idx: string]: Author } = {
...
       "Mike is a software architect who values developer experience, productivity and team morale. He is on a journey to help developers let go of their local development environments and embrace automated, ephemeral development environments.",
   },
+  arthursens: {
+    name: "Arthur Sens",
+    socialProfiles: {
+      github: "arthursens",
+      twitter: "ArthurSilvaSens",
+      linkedin: "arthursilvasens",
+    },
+    description: "",
```
Comment (mikenikles) [inline src/contents/blog.ts:156]: @ArthurSens please provide a short bio. It's not currently used anywhere, but for the sake of consistency, let's add it 😄.

File: src/contents/blog.ts [hunk 3]
```diff
@@ -146,4 +146,13 @@ export const authors: { [idx: string]: Author } = {
     description:
       "Mike is a software architect who values developer experience, productivity and team morale. He is on a journey to help developers let go of their local development environments and embrace automated, ephemeral development environments.",
   },
+  arthursens: {
+    name: "Arthur Sens",
+    socialProfiles: {
+      github: "arthursens",
+      twitter: "ArthurSilvaSens",
+      linkedin: "arthursilvasens",
```
Comment (mikenikles) [inline src/contents/blog.ts:154]: Please double check that I picked the correct usernames.

File: src/routes/blog/journey-to-site-reliability-engineering-at-gitpod.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
+---
+author: arthursens
+date: Sunday, 23 May 2021 11:00:00 UTC
+excerpt: The SRE team at Gitpod is making the first moves to implement Site Reliability Engineering best practices. We document our approach to implement our very first Service Level Objective.
+image: teaser.jpg
+slug: journey-to-site-reliability-engineering-at-gitpod
+subtitle:
+teaserImage: teaser.jpg
```
Comment (mikenikles) [inline src/routes/blog/journey-to-site-reliability-engineering-at-gitpod.md:8]: To fix the build, and allow this PR to get merged, we need a `teaser.jpg` file in the `static/images/blog/journey-to-site-reliability-engineering-at-gitpod` directory. This image is displayed on the blog landing page (`/blog`) and at the top of the blog post itself.

File: src/routes/blog/journey-to-site-reliability-engineering-at-gitpod.md (hunk 1 above)
Comment (mikenikles) [reply src/routes/blog/journey-to-site-reliability-engineering-at-gitpod.md:24]: If you want to mention that, how about something like this?

Gitpod is scaling! [We've recently announced that we got $$$$](https://www.gitpod.io/blog/next-chapter-for-gitpod/) - we are glad to be backed by fantastic investors - and the topic ...

**Review by mikenikles — APPROVED** [review]
This is a great blog post Arthur 🔥!

I left some instructions to fix the build. Once done and the Netlify build passes, you can go ahead and merge this. It will be online a few minutes after the PR gets merged.

If you run into any issues, drop me a note ✍️.

File: src/contents/blog.ts (hunk 3 above)
Comment (ArthurSens) [reply src/contents/blog.ts:154]: That is correct, thanks!

**Comment by ArthurSens** [comment]
Thanks for the review @mikenikles! Everything seems to be fixed now 🙂 

Should I just merge then?

**Comment by mikenikles** [comment]
Fantastic, yep let's merge it 🎉

**Review by meysholdt — APPROVED** [review]
Great article! Green light from my side to publish this, too!

**Pull request merged**